src/iocore/eventsystem/MIOBufferWriter.cc

```diff
@@ -76,7 +76,7 @@ MIOBufferWriter::operator>>(std::ostream &stream) const
 }
 
 ssize_t
-MIOBufferWriter::operator>>(int fd) const
+MIOBufferWriter::operator>>(int /* fd ATS_UNUSED */) const
 {
   return 0;
 }
```

Comment:
Um... interesting. Can this whole method be removed I wonder?

Reply:
I'm not sure about this one.
It's inside
```
#if defined(UNIT_TEST_BUFFER_WRITER)
...
#else
```
and in the `#else` branch this same method has an actual implementation.

Reply:
Sure enough, it's for the unit test. Let's leave this one as you have it for now.